docs/installation/config/env_config.rst

```diff
@@ -0,0 +1,284 @@
+  file uploads.
+
+* ``DOCUMENTEN_UPLOAD_READ_CHUNK``: chunk size in bytes for large file uploads - when
+  merging upload chunks, this determines the number of bytes read to copy to the
+  destination file. Defaults to 6 MiB.
+
+* ``SENDFILE_BACKEND``: which backend to use for authorization-secured upload
+  downloads. Defaults to ``sendfile.backends.nginx``. See
+  [django-sendfile2](https://pypi.org/project/django-sendfile2/) for available
+  backends.
+
+* ``SENTRY_DSN``: URL of the sentry project to send error reports to. Default
+  empty, i.e. -> no monitoring set up. Highly recommended to configure this.
+
+* ``JWT_EXPIRY``: duration a JWT is considered to be valid, in seconds. Defaults to 3600 -
+  1 hour.
+
+* ``JWT_LEEWAY``: JWT validation has a time aspect, usually in the form of the ``iat`` and
+  ``nbf`` claims. Clock drift between server and client can occur. This setting allows
+  specifying the leeway in seconds, and defaults to ``0`` (no leeway). It is advised to
+  not make this larger than a couple of minutes.
+
+* ``LOG_STDOUT``: whether to log to stdout or not. For Docker environments, defaults to
+  ``True``, for other environments the default is to log to file.
+
+* ``LOG_QUERIES``: enable (query) logging at the database backend level. Note that you
+  must also set ``DEBUG=1``, which should be done very sparingly!
+
+* ``LOG_LEVEL``: control the verbosity of logging output. Available values are ``CRITICAL``,
+  ``ERROR``, ``WARNING``, ``INFO`` and ``DEBUG``. Defaults to ``WARNING``.
+
+* ``LOG_REQUESTS``: enable logging of the outgoing requests. Defaults to ``False``.
+
+* ``PROFILE``: whether to enable profiling-tooling or not. Applies to the development
+  settings only. Defaults to ``False``.
+
+* ``CMIS_ENABLED``: whether to enable the CMIS adapter. Defaults to ``False``.
+
+* ``CMIS_MAPPER_FILE``: name of the file containing the mapping between the Django and Document Management System names
+    for document properties. See the installation section for more details.
+    Defaults to the absolute path of ``open-zaak/config/cmis_mapper.json``.
+
+* ``CMIS_URL_MAPPING_ENABLED``: enable the URL shortener when using the CMIS adapter.
+  Defaults to ``False``.
+
+* ``EXTRA_VERIFY_CERTS``: a comma-separated list of paths to certificates to trust, empty
+  by default. If you're using self-signed certificates for the services that Open Zaak
+  communicates with, specify the path to those (root) certificates here, rather than
+  disabling SSL certificate verification. Example:
+  ``EXTRA_VERIFY_CERTS=/etc/ssl/root1.crt,/etc/ssl/root2.crt``.
+
+* ``CURL_CA_BUNDLE``: if this variable is set to an empty string, it disables SSL/TLS certificate
+    verification. Even calls from Open Zaak to other services 
+    such as the [Selectie Lijst](https://selectielijst.openzaak.nl/) will be disabled, so this
+    variable should be used with care to prevent unwanted side-effects.
```

Comment:
Same comment as above
```suggestion
* ``CURL_CA_BUNDLE``: if this variable is set to an empty string, it disables SSL/TLS certificate
  verification. Even calls from Open Zaak to other services 
  such as the [Selectie Lijst](https://selectielijst.openzaak.nl/) will be disabled, so this
  variable should be used with care to prevent unwanted side-effects.
```